src/explorer-link.ts

```diff
@@ -2,5 +2,5 @@ import prefixForNetwork from './prefix-for-network'
 
 export = function getExplorerLink(hash: string, network: string): string {
   const prefix = prefixForNetwork(network)
-  return `https://${prefix}etherscan.io/tx/${hash}`
+  return (prefix !== 'other.' ?`https://${prefix}etherscan.io/tx/${hash}`: '');
```

Comment:
Nit: These parentheses aren't necessary:
```suggestion
  return prefix !== 'other.' ?`https://${prefix}etherscan.io/tx/${hash}`: '';
```